web/server/src/soro_server.cc

```diff
@@ -132,8 +138,19 @@ void serve_file(std::string const& decoded_url,
 
 bool serve_tile(server::serve_context& sc, std::string const& decoded_url,
                 auto const& req, auto& res) {
-  static tiles::regex_matcher const matcher{R"(\/(\d+)\/(\d+)\/(\d+).mvt)"};
-  auto const match = matcher.match(decoded_url);
+
+
```

Comment:
This code appears to be from an older version of the branch OSM_Visualisation and should probably not be merged into the master since it failed in a code review before